CustomerJourney/package.json

```diff
@@ -26,7 +26,7 @@
     "lit-html": "1.2.1"
   },
   "peerDependencies": {
-    "@cjaas/common-components": "4.0.1",
+    "@cjaas/common-components": "4.1.1-dev1",
```

Comment:
just make sure to remember to update this to match a real version 4.1.1 instead of dev version once common-components is published.